data/Ernesto.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Ernesto",
+  "url": "https://www.ernesto.it/",
+  "career_page_url": "https://help.ernesto.it/contatti/",
```

Comment:
This page is not a valid career page, do you have any alternative?

Reply:
I do not unfortunately, candidates can send us a plain email and get in touch